recipes-ni/ni-systemimage/ni-systemimage.bb

```diff
@@ -23,9 +28,12 @@ do_install () {
 }
 
 
-FILES_${PN} += "\
+FILES:${PN} += "\
 	${bindir}/nisystemimage \
 	/usr/local/natinst/bin/nisystemimage \
 "
-
-RDEPENDS_${PN} += "bash ni-netcfgutil"
+RDEPENDS:${PN} += "\
+	bash \
+	ni-netcfgutil \
+	niacctbase \
```

Comment:
Added dependency; why?

Reply:
The `do_install()` uses the `${LVRT_GROUP}` variable - which is only meaningful when the `niacctbase` package is installed. `niacctbase` sets up the LVRT user, group, and home directory.

Reply:
It'd be nice if you could call that out in the commit description and explain why.